package.json

```diff
@@ -74,7 +79,29 @@
     "esbuild": "^0.24.0",
     "eslint": "^8.57.1",
     "npm-run-all": "^4.1.5",
-    "typescript": "^5.6.3"
+    "typescript": "^5.6.3",
+    "jest": "^29.0.0",
+    "jest-environment-jsdom": "^29.7.0",
+    "jest-junit": "^14.0.0",
+    "ts-jest": "^29.2.5"
+  },
+  "jest": {
+    "preset": "ts-jest",
+    "testEnvironment": "jsdom",
+    "transform": {
+      "^.+\\.tsx?$": "ts-jest"
+    },
+    "moduleFileExtensions": [
+      "ts",
+      "tsx",
+      "js",
+      "jsx",
+      "json",
+      "node"
+    ],
+    "moduleNameMapper": {
+      "\\.(css|less|scss|sass)$": "identity-obj-proxy"
+    }
```

Comment:
those changes seem unnecessary, since the tests only affect the webviews/ package